.github/workflows/ci.yml

```diff
@@ -106,4 +110,31 @@ jobs:
         # Can't use `go test -v ./...` because some of the tests depend on SEV-SNP or Confidential ACI features
         go test -v ./... -tags skip_e2e,skip_snp_required
 
+  test-attestation:
```

Comment:
Do you think each sidecar should be its own job or should we have tests for each sidecar under this job to avoid duplicating the steps of installing the testing package?

Reply:
I think the benefit of having each sidecars test running independently is worth the bit of duplicated setup?